### ISSUE TITLE
Odd warnings due unsupported rules when extending a config

### ISSUE THREAD

**Issue by lucasmazza** [codeclimate/codeclimate-eslint#214]
When extending the `airbnb` config, none of the `eslint-plugin-import` rules are found:

```
$  codeclimate  analyze
Starting analysis
Running eslint: Done!

== index.js (17 issues) ==
1: Definition for rule 'import/no-duplicates' was not found [eslint]
1: Definition for rule 'import/no-unresolved' was not found [eslint]
1: Definition for rule 'import/no-named-as-default' was not found [eslint]
1: Definition for rule 'import/no-named-as-default-member' was not found [eslint]
1: Definition for rule 'import/no-extraneous-dependencies' was not found [eslint]
1: Definition for rule 'import/no-mutable-exports' was not found [eslint]
1: Definition for rule 'import/no-amd' was not found [eslint]
1: Definition for rule 'import/first' was not found [eslint]
1: Definition for rule 'import/no-named-default' was not found [eslint]
1: Definition for rule 'import/extensions' was not found [eslint]
1: Definition for rule 'import/newline-after-import' was not found [eslint]
1: Definition for rule 'import/prefer-default-export' was not found [eslint]
1: Definition for rule 'import/no-absolute-path' was not found [eslint]
1: Definition for rule 'import/no-dynamic-require' was not found [eslint]
1: Definition for rule 'import/no-webpack-loader-syntax' was not found [eslint]
1: Definition for rule 'import/export' was not found [eslint]
1: 'React' is defined but never used. [eslint]
```
Adding `plugins: ['import']` to my config raises another warnings for different rules from the airbnb config. Locally this doesn't happen when the image is rolledback to the **b457** tag (the previous commit from the #204 merge).

[Sample repository here.](https://github.com/lucasmazza/eslint-extends-test)

**Comment by ahoym** [comment]
I'm actually seeing something similar as well. Here's a screenshot I took while using the codeclimate chrome plugin:

<img width="710" alt="screen shot 2017-03-16 at 2 28 55 pm" src="https://cloud.githubusercontent.com/assets/5249615/24019668/261a7718-0a56-11e7-957f-bdf458929ae6.png">

Which pops up in almost every file that adds or modifies an `import`

**Comment by dblandin** [comment]
@lucasmazza @ahoym We're on it. Thank you for the detailed issue and example repo!

Looks like #204 may be interfering with our vendored plugins. Working on a fix now.

**Comment by dblandin** [comment]
@lucasmazza I just released an engine update ([b500](https://github.com/codeclimate/codeclimate-eslint/releases/tag/b500)) including the revert in #216. I verified the revert locally using the CLI and the example repo you provided (thanks again!).

We'll use that going forward as we re-introduce the updates desired in #204.

Let me know when you're able to verify the behavior on your end.

Thanks!

**Comment by filipesperandio** [comment]
@lucasmazza After some tuning this is the out put I get for the sample repo:
```
$ codeclimate analyze
Starting analysis
Running eslint: Done!

== index.js (3 issues) ==
1: 'React' is defined but never used. [eslint]
1: Unable to resolve path to module 'react'. [eslint]
1: Missing file extension for "react" [eslint]

Analysis complete! Found 3 issues.
```

Makes sense?

**Comment by dblandin** [comment]
We should add those last two checks to a blacklist or and either prevent the check from running altogether or prevent the engine from generating an issue. Checks that rely on module resolution will not work properly on production since we don't install dependencies prior to analysis.

```
import/no-unresolved
import/extensions
```

**Comment by lucasmazza** [comment]
> Makes sense?

Yes :+1:

**Comment by lucasmazza** [comment]
@filipesperandio does that output comes from the branch from #215?

**Comment by filipesperandio** [comment]
@lucasmazza Will be on that branch soon. Still polishing the code and running other tests.

**Comment by filipesperandio** [comment]
Fixed by 55c8649

**Comment by filipesperandio** [comment]
@lucasmazza Container tag *b533* has the fix
